type/__clean_path/man.rst

```diff
@@ -3,38 +3,42 @@ cdist-type__clean_path(7)
 
 NAME
 ----
-cdist-type__clean_path - Remove files and directories which match the pattern.
+cdist-type__clean_path - Remove files and directories in path recursively.
 
 
 DESCRIPTION
 -----------
-Remove files and directories which match the pattern.
-
 Provided path must be a directory.
 
-Patterns are passed to ``find``'s ``-regex`` - see ``find(1)`` for more details.
-
-Look up of files and directories is non-recursive (``-maxdepth 1``).
+Arguments are passed to ``find(1)``.
 
 Parent directory is excluded (``-mindepth 1``).
 
-This type is not POSIX compatible (sorry, Solaris users).
```

Comment:
I think the `-not-*` should be fixable for POSIX, but `-*-regex` is not POSIX for sure*.

-----
\* unless maybe with an `-exec` hack 🤔

Reply:
Marked as not POSIX in man for now. You can look into this `-exec` hack later :smile: